android/src/main/java/com/zmxv/RNSound/RNSoundModule.java

```diff
@@ -576,19 +578,9 @@ public void getCurrentTime(final Double key, final Callback callback) {
     callback.invoke(player.getCurrentPosition() * .001, player.isPlaying());
   }
 
-  //turn speaker on
   @ReactMethod
-  public void setSpeakerphoneOn(final Double key, final Boolean speaker) {
-    MediaPlayer player = this.playerPool.get(key);
-    if (player != null) {
-      AudioManager audioManager = (AudioManager)this.context.getSystemService(this.context.AUDIO_SERVICE);
-      if(speaker){
-        audioManager.setMode(AudioManager.MODE_IN_COMMUNICATION);
-      }else{
-        audioManager.setMode(AudioManager.MODE_NORMAL);
-      }
-      audioManager.setSpeakerphoneOn(speaker);
-    }
+  public void setSpeakerphoneOn(final Boolean forcePhoneSpeaker) {
```

Comment:
Could you perhaps rename this from `setSpeakerPhoneOn` to something like `forcePhoneSpeaker`?

SpeakerPhone is term related to audio routing during a phone call (and what the original implementation was setting up) and has a different meaning than what we now use it to even though both are kind of trying to achieve a similar result moving to play over the speaker while another output is active the context is a bit different as we are not doing this during a call.

Reply:
Yes I can ;)